src/data/finset/basic.lean

```diff
@@ -1743,6 +1745,18 @@ end
   to_finset l.reverse = l.to_finset :=
 to_finset_eq_of_perm _ _ (reverse_perm l)
 
+lemma disjoint_to_finset_iff_disjoint {l l' : list α} :
+  _root_.disjoint l.to_finset l'.to_finset ↔ l.disjoint l' :=
```

Comment:
The `multiset` version is at the end of the file, could you put this next to that for consistency? There's a list namespace already and this would fit nicely there.